### ISSUE TITLE
Fix translations of the preview toggle buttons aria-label

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#1035]
The preview toggle buttons use `aria-label` attributes but their translation should be fixed, the js template references wrong property names in the i18n object.

**Comment by atimmer** [comment]
Closed in #1036
